.github/workflows/release.yml

```diff
@@ -0,0 +1,92 @@
+name: Create GitHub Release
```

Comment:
```suggestion
name: Build and publish AppImages
```

Since this is no longer exclusive to releases.

Reply:
Done.